charts/backbeat/templates/gc/deployment.yaml

```diff
@@ -40,10 +40,12 @@ spec:
             httpGet:
               path: {{ .Values.health.path.liveness}}
               port: {{ .Values.health.port }}
+            initialDelaySeconds: 10
```

Comment:
Since most of the backbeat deployments share this value, maybe it should be specified in the `values.yaml` file. Then it could be templated with something like:
```yaml
        	livenessProbe:
{{ toYaml .Values.livenessProbe | indent 12 }}
        	readinessProbe:
{{ toYaml .Values.readinessProbe | indent 12 }}
```

Reply:
For now I keep those changes as an experiment to see how it goes on the CI, but if it helps I'll do this change!